packages/node_modules/@node-red/editor-client/src/js/ui/tab-info-outliner.js

```diff
@@ -362,6 +362,9 @@ RED.sidebar.info.outliner = (function() {
         var existingObject = objects[n.id];
         var parent = n.g||n.z;
 
+        if (!existingObject) {
+            return;
+        }
```

Comment:
See my main issue comment. This is a sensible check to make, but in reality, the 'change' event should *not* be triggered by deleting a group.